app/models/case_log.rb

```diff
@@ -431,6 +432,15 @@ def get_totadult
     end
   end
 
+  def get_refused
+    return 1 if [age1_known, age2_known, age3_known, age4_known, age5_known, age6_known, age7_known, age8_known].any?(1)
```

Comment:
Technically this is "Don't know" rather than "Tenant prefers not to say" (which == refused) right? Should we add an option for that? cc. @paulrobertlloyd